tools/code_analyzer/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(code_analyzer)
+
+find_package(LLVM REQUIRED CONFIG)
+add_definitions(${LLVM_DEFINITIONS})
+include_directories(${LLVM_INCLUDE_DIRS})
+link_directories(${LLVM_LIBRARY_DIRS})
+
+add_library(OpDependencyPass MODULE
+  op_dependency.cpp
+)
+
+set_target_properties(OpDependencyPass PROPERTIES
+    COMPILE_FLAGS "-std=c++11 -fno-rtti -O3")
```

Comment:
Since we just switched pytorch to c++14, it's probably better to use it her too. Also, the `CXX_STANDARD` target property is a better way of setting it.